utils/produce_insights_results/produce.sh

```diff
@@ -19,7 +19,7 @@ do
     for file in *.json
     do
         echo $file
-        kafkacat -b localhost:9092 -P -t ccx.ocp.results $file
+        kafkacat -b localhost:29092 -P -t ccx.ocp.results $file
```

Comment:
If we gonna use it with our `docker-compose`, then I think we should update configuration of consumer as well. I mean change `address = "localhost:9092"` -> `address = "localhost:29092"`

Reply:
@bond95 @quarckster I was just about to raise the question here if I should remove this or update atleast the `config-devel.toml`. Or even in other configs since in other envs it's set by `KAFKA_BOOTSTRAP_PORT`, right? I think everyone runs the `docker-compose` now but I'm not sure about other configs

Reply:
We should update all config for consistensy sake.

Reply:
Agree

Reply:
Done

Reply:
Thanks for this! It's quite important that the parts integrate well together. Finding various misconfigured ports is a nightmare.